README.md

```diff
@@ -75,6 +75,24 @@ Note, you can use any of the supported [storage backends](https://restic.readthe
    ````
 1. (optional) Setup email on failure as described [here](#8-email-notification-on-failure)
 
+# 🏃 Restic wrapper
+
+For convenience there's a `restic` wrapper script that makes loading profiles and **running restic**
+straightforward (it needs to run with sudo to read environment). Just run:
+
+- `sudo resticw WHATEVER` (e.g. `sudo resticw snapshots`) to use the default profile.
+- You can run the wrapper by passing a specific profile: `resticw -p anotherprofile snapshots`.
+
+### Useful commands
+
+| Command                                           | Description                                                       |
+|---------------------------------------------------|-------------------------------------------------------------------|
+| `resticw snapshots`                               | List backup snapshots                                             |
+| `resticw diff <snapshot-id> latest`               | Show the changes from the latest backup                           |
+| `resticw stats` / `resticw stats snapshot-id ...` | Show the statistics for the whole repo or the specified snapshots |
+| `resticw mount /mnt/restic`                       | Mount your remote repository                                      |
+
+
```

Comment:
To make this easy to maintain, let's add a bullet point in the section `# Development`, that lists the minimal commands to update the parser

```console
$ pip install doctopt.sh
$ doctopt.sh usr/local/sbin/resticw
```

or something. It's very useful to have these short command notes down the line when adding a new paramter in a few years.. :)

Reply:
Sure, done!